Games/types/Mafia/items/Armor.js

```diff
@@ -30,9 +36,13 @@ module.exports = class Armor extends Item {
     }
 
     hold(player) {
-        for (let effect of player.effects) {
-            if (effect.name == "Armor") {
-                effect.uses++;
+        for (let item of player.items) {

```

Comment:
should iterate items instead of effects, see 
https://github.com/r3ndd/BeyondMafia-Integration/blob/f8efab54f0ec652ae2c03afe1ac20a12c9c8fdb4/Games/types/Mafia/items/CursedArmor.js#L12